.devcontainer/devcontainer.json

```diff
@@ -27,11 +27,12 @@
 			}
 		}
 	},
+	"features": {
```

Comment:
Do we really need this? It is already installed into the base container.

Reply:
Good observation, it is not needed anymore, I have deleted it.